src/test/java/parser/JsonParserTest.java

```diff
@@ -0,0 +1,52 @@
+package parser;
+
+import org.junit.jupiter.api.*;
+import shop.Cart;
+
+import java.io.File;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class JsonParserTest {
```

Comment:
please use access modifiers in class, fields and methods

Reply:
Done